backend/aws/lambda/database_calls.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+from botocore.exceptions import ClientError
+
+class database_calls():
```

Comment:
Class names should be `CapWords` convention, source: https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Good catch. Thank you!